setup.py

```diff
@@ -40,6 +53,51 @@
     'nose>=1.0',
 ]
 
+class CustomCommands(setuptools.Command):
+  """A setuptools Command class able to run arbitrary commands."""
+
+  def initialize_options(self):
+    pass
+
+  def finalize_options(self):
+    pass
+
+  def RunCustomCommand(self, command_list):
+    print 'Running command: %s' % command_list
+    p = subprocess.Popen(
```

Comment:
Does python not have a simpler exec form when we don't care about using the standard input?

Reply:
Done, but error handling is worse this way.